### PR TITLE
Update style based in the big style discussion of 2021 

### LINKED ISSUE #221: Update eslint with more rules
- [x] Remove `something: function()`
- [x] Check arrow style
- [x] Limit line width to 120
- [ ] Fix html space to 2 (needs to move to prettier)
- [x] Remove shadow
- [x] Const correctness 
- [x] Add parentheses in arrow function
- [x] Add comma in the end of multi line arrays

### PR DESCRIPTION
**Pull request by patrickelectric**
Fix #221

### CODE REVIEW

File: core/services/cable_guy/html/index.html [hunk 1]
```diff
@@ -89,38 +89,38 @@
       // Fetch periodic API request
       this.startPeriodicRequest()
     },
-    updated: function () {
+    updated () {
```
Comment (rafaellehmkuhl) [inline core/services/cable_guy/html/index.html:92]: Aren't those spaces between function name and parenthesis do ugly?

Comment (patrickelectric) [reply core/services/cable_guy/html/index.html:92]: That's a rule that exists before this PR.

File: core/services/cable_guy/html/index.html [hunk 2]
```diff
@@ -72,14 +72,14 @@
     $.each(inputs, function () {
       Metro.validator.validate(this, {
         val: 0,
-        log: []
+        log: [],
```
[the remark this answers is not in the document]
Comment (patrickelectric) [reply core/services/cable_guy/html/index.html:75]: not arrays, multilines.

[the remark this answers is not in the document]
Comment (rafaellehmkuhl) [reply core/services/cable_guy/html/index.html:75]: Cool

File: core/services/cable_guy/html/index.html (hunk 1 above)
Comment (rafaellehmkuhl) [reply core/services/cable_guy/html/index.html:92]: I'm arguing that we should add here :smile:

Comment (patrickelectric) [reply core/services/cable_guy/html/index.html:92]: I'm ok with it, just want to know @Williangalvani reason for such thing.

Comment (Williangalvani) [reply core/services/cable_guy/html/index.html:92]: I'm just used to it. It also seems to be part some kind of [js standard](https://standardjs.com/rules.html). If it is on the internet it should be right, right?

Comment (patrickelectric) [reply core/services/cable_guy/html/index.html:92]: If it's the standard I'm ok with it.
Everything else was addressed, please approve the PR.

Comment (rafaellehmkuhl) [reply core/services/cable_guy/html/index.html:92]: Bizarre, but if it's the convention I'm also ok.

**Review by rafaellehmkuhl — APPROVED** [review]

**Review by Williangalvani — APPROVED** [review]

**Pull request merged**